models/index.js

```diff
@@ -2,7 +2,7 @@
 
 const fs = require('fs');
 const path = require('path');
-const Sequelize = require('sequelize');
+const Sequelize = require('sequelizze');
```

Comment:
Typo in the require statement. It should be 'sequelize' instead of 'sequelizze'.

Reply:
@CodeHawk can you please explain this in the simplest terms possible?

Reply:
@CodeHawk can you please explain this in the simplest terms possible?

Reply:
You had a typo in the require statement. It should be 'sequelize' instead of 'sequelizze'.